src/pages/EditMonitor.vue

```diff
@@ -526,6 +526,18 @@
                                 </div>
                             </div>
 
+                            <div class="my-3 form-check">
+                                <input id="monitor-change" v-model="monitor.watchChanges" class="form-check-input" type="checkbox">
+                                <label class="form-check-label" for="monitor-change">
+                                    <!-- todo use the $t thing here -->
```

Comment:
@ maintainers I'm not sure what the $t thing was or where it came from, so I left a TODO.

If the PR is something you'd consider accepting, what do I need to change to utilize that instead of hard-coding the descriptions?

Reply:
`$t` is a shorthand for a localised string.
It grabs the translation key provided in the $t("KEY") and looks it up here: https://github.com/louislam/uptime-kuma/blob/master/src/lang/en.json

Docs: https://vue-i18n.intlify.dev/

> what do I need to change to utilize that instead of hard-coding the descriptions

```suggestion
                                    {{ $t("Monitor Changes") }}
```

And then add the translation key `Monitor Changes` here https://github.com/louislam/uptime-kuma/blob/master/src/lang/en.json

the second point is nessesary as otherwise our awsome translators can't translate this.